src/histolab/tiler.py

```diff
@@ -578,28 +582,39 @@ def _highest_score_tiles(self, slide: Slide) -> List[Tuple[float, CoordinatePair
         List[Tuple[float, CoordinatePair]]
             List of tuples containing the score and the extraction coordinates for the
             tiles with the highest score. Each tuple represents a tile.
+        List[Tuple[float, CoordinatePair]]
+            List of tuples containing the scaled score between 0 and 1 and the
+            extraction coordinates for the tiles with the highest score. Each tuple
+            represents a tile.
```

Comment:
??? 2 lists?

Reply:
yes because now we have to consider also the scaled version of the scores (from 0 to 1)